CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 22.4.0
 
+### Breaking Changes
+- Upgrade published docker images to latest Ubuntu LTS. This may require docker runtimes to be upgraded as well.  
```

Comment:
suggestion: `As Web3Signer docker image uses Ubuntu latest tag, it may require manual update of latest container runtime specially if host environment is other than Ubuntu and/or automated updates are not enabled. See Ubuntu [bug](https://bugs.launchpad.net/ubuntu/+source/libseccomp/+bug/1916485) for more details.